### ISSUE TITLE
lib/hyperspec.el: Make the variables `common-lisp-hyperspec-root` and `common-lisp-hyperspec-symbol-table` actually customizable

### ISSUE THREAD

**Issue by ackalker** [slime/slime#245]
The help text for `common-lisp-hyperspec` has the following suggestion:

"[...] If you copy the HyperSpec to another location, customize the variable
`common-lisp-hyperspec-root' to point to that location. [...]"

but the variable isn't actually customizable because it is declared using `defvar`.
I think the same applies to `common-lisp-hyperspec-symbol-table` as well.

**Comment by luismbo** [comment]
I guess they're "customizable" via `setq`. :-) Your idea of declaring these variable using `defcustom` makes perfect sense. Do you want to prepare a pull request doing that?

**Comment by ackalker** [comment]
True, although using plain `setq` leaves one with those pesky warnings (by an otherwise very useful on-the-fly code checker) about undefined free variables in your init file, until one realizes that
one should rather use `defvar` oneself :-)

If I have some time, i'll gladly make a pull request.
